spec/controllers/users_controller_spec.rb

```diff
@@ -203,6 +203,13 @@
         expect(Workers::Mail::ConfirmEmail).to receive(:perform_async).with(@user.id).once
         put(:update, :id => @user.id, :user => { :email => "my@newemail.com"})
       end
+
+      it "saves unconfirmed_email when podmail is disabled" do
+        AppConfig.mail.enable = false
+        put(:update, :id => @user.id, :user => { :email => "my@newemail.com"})
```

Comment:
Please fix this code style issue. The hash syntax should be 

```ruby
put(:update, id: @user.id, user: {...})
```

You can check the code style locally as described here: https://wiki.diasporafoundation.org/Styleguide#Automatic_local_review